rt/transports/http-netty/netty-client/src/test/resources/org/apache/cxf/transport/http/netty/client/integration/ServerConfig.xml

```diff
@@ -31,7 +31,7 @@
     <bean class="org.springframework.context.support.PropertySourcesPlaceholderConfigurer"/>
     <httpj:engine-factory>
         <httpj:engine port="${SSLNettyClientTest.port}">
-            <httpj:tlsServerParameters>
+            <httpj:tlsServerParameters sniHostCheck="false">
```

Comment:
`false` is the default, right?

Reply:
@reta  Sorry for long delay, I  missed this comment. Yes, false is the default for backward compatibility.

Reply:
np at all, :+1: